src/main/java/net/glowstone/chunk/GlowChunk.java

```diff
@@ -242,7 +246,7 @@ public void initializeSections(ChunkSection... initSections) {
         }
         //GlowServer.logger.log(Level.INFO, "Initializing chunk ({0},{1})", new Object[]{x, z});
 
-        sections = new ChunkSection[DEPTH / SEC_DEPTH];
+        sections = new ChunkSection[SEC_COUNT];
         System.arraycopy(initSections, 0, sections, 0, Math.min(sections.length, initSections.length));
```

Comment:
I'm not entirely sure what that would look like.  Do you mean replacing `initializeSections(ChunkSection... initSections)` with `initializeSection(int index, ChunkSection section)`?  There might be _some_ benefit, but I don't know where that would be used (all places where `initializeSections` are used already build an array).  It would get rid of the array with null members being passed, though.